pom.xml

```diff
@@ -60,9 +60,8 @@
 		<releaseExt>-preview</releaseExt>
 
 		<!-- Driver Dependencies -->
-		<azure.keyvault.version>1.2.4</azure.keyvault.version>
-		<azure.adal4j.version>1.6.5</azure.adal4j.version>
-		<rest.client.version>1.7.4</rest.client.version>
+		<azure.keyvault.version>4.2.1</azure.keyvault.version>
```

Comment:
When I load the project in Eclipse, I get  dependency conflicts. @lilgreenbird  I thought you had a fix for it, but can't see it on the PR?